python/tvm/relay/frontend/tflite.py

```diff
@@ -747,6 +749,158 @@ def convert_squared_difference(self, op):
         out = _op.power(difference, relay.const(2, exp_type))
         return out
 
+    def convert_gather(self, op):
+        """Method to Convert TFLite Gather operator"""
+        # Check if the input tensor is quantized, call QNN op
+        if self.is_quantized(op):
+            raise tvm.error.OpNotImplemented(
+                'TFlite quantized gather operator is not supported yet.')
```

Comment:
Since this is a tensor manipulation operation rather than something that requires any specific qnn dialect, I would prefer we handle the qnn ops here.